recipe/bld.bat

```diff
@@ -0,0 +1,18 @@
+mkdir build
+cd build
+if %ERRORLEVEL% neq 0 exit 1
+
+cmake -GNinja ^
+    -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+    ..
```

Comment:
```suggestion
:: Generate the build files.
echo "Generating the build files..."
cmake .. %CMAKE_ARGS% ^
      -G"Ninja" ^
      -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
      -DCMAKE_BUILD_TYPE=Release
```

Reply:
It's from the template https://github.com/anaconda-distribution/distro-incubator/tree/main/skel